### PR TITLE
fix case where the user has no git config

### PR DESCRIPTION
**Pull request by 0atman**
Accidentally found this out after my 20.04 reinstall last week!

### CODE REVIEW

File: register.py [hunk 1]
```diff
@@ -17,6 +18,10 @@ def register(mode="prod"):
     username = run("git config --global user.name")
     email = run("git config --global user.email")
 
+    if not username:
```
Comment (akinnane) [inline register.py]:
```suggestion
    if not username or not email:
```
Do we need to check for an invalid email too?

**Review by akinnane — CHANGES_REQUESTED** [review]

File: register.py (hunk 1 above)
Comment (0atman) [reply register.py]: yes, yes we do

**Comment by 0atman** [comment]
@akinnane look good now?

**Review by akinnane — APPROVED** [review]
Great, thanks.

**Pull request merged**